src/isomorphic/modern/element/__tests__/ReactJSXElementValidator-test.js

```diff
@@ -101,19 +103,34 @@ describe('ReactJSXElementValidator', () => {
       },
     };
 
-    void <Component>{iterable}</Component>;
+    ReactTestUtils.renderIntoDocument(<Component>{iterable}</Component>);
 
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(console.error.calls.argsFor(0)[0]).toContain(
       'Each child in an array or iterator should have a unique "key" prop.',
     );
   });
 
+  it('does not warns for fragments of multiple elements without keys', () => {
+    spyOn(console, 'error');
+
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
Looks like your PR branch has fallen behind master. This flag no longer exists since we deleted Stack. You'll want to rebase it.